satellite/metainfo/loop.go

```diff
@@ -188,6 +189,11 @@ waitformore:
 					return LoopError.New("unexpected error unmarshalling pointer %s", err)
 				}
 
+				//if RepairOverride is set, ensure the pointer contains the new value
+				if loop.config.RepairOverride != 0 {
+					pointer.Remote.Redundancy.RepairThreshold = int32(loop.config.RepairOverride)
+				}
+
```

Comment:
My only concern here is that this only changes the value in memory. What if future code pulls the pointer from the DB before acting on it.  Not sure this is a real concern, just thought I'd mention it.

Reply:
I am restructuring the code to handle this directly in the checker and repairer :)